modules/bitgo/test/v2/unit/internal/tssUtils/ecdsa.ts

```diff
@@ -909,8 +911,8 @@ describe('TSS Ecdsa Utils:', async function () {
         { name: 'commonKeychain', value: bitgoCombined.xShare.y + bitgoCombined.xShare.chaincode },
         { name: 'userKeyId', value: userKeyId },
         { name: 'backupKeyId', value: backupKeyId },
-        { name: 'bitgoToUserPublicShare', value: userToBitgoShare.publicShare },
-        { name: 'bitgoToBackupPublicShare', value: backupToBitgoShare.publicShare },
+        { name: 'bitgoToUserPublicShare', value: userToBitgoShare.privateShareProof as string },
```

Comment:
This `privateShareProof` is generated via `createShareProof()` in `opengpgutils`:
https://github.com/BitGo/BitGoJS/blob/master/modules/sdk-core/src/bitgo/utils/opengpgUtils.ts#L170